app/src/components/WithAdminPermissions/AdminPermissions.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export class WithAdminPermissions extends React.Component {
+  componentDidMount() {
+    if (!this.props.isLoggedIn) {
+      this.props.redirectToLogin();
```

Comment:
This redirection logic is duplicated from `WithPermissions`. If keeping separate, is this redirection appropriate here?